pull_request_template.md

```diff
@@ -2,7 +2,8 @@
 
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
-Fixes # (issue)
+Fixes # (issue)</br>
+Files Changed:
```

Comment:
Actually we don't need to list files changes in the description as it's already part of the PR content.
Instead of file changes, maybe we can add "Screenshot" if any because this project is mostly frontend so most changes will affect UI and a screenshot is very helpful.

Reply:
Fixed